cmd/admin-heal.go

```diff
@@ -153,8 +153,12 @@ func (s backgroundHealStatusMessage) String() string {
 	healPrettyMsg := console.Colorize("HealBackgroundTitle", "Background healing status:\n")
 	healPrettyMsg += fmt.Sprintf("  Total items scanned: %s\n",
 		console.Colorize("HealBackground", s.HealInfo.ScannedItemsCount))
-	healPrettyMsg += fmt.Sprintf("  Last background heal check: %s\n",
-		console.Colorize("HealBackground", timeDurationToHumanizedDuration(time.Since(s.HealInfo.LastHealActivity)).String()+" ago"))
+	lastHealingTime := "Never"
```

Comment:
It is better to show when it is anticipated to run too..

Reply:
> It is better to show when it is anticipated to run too..

Done